packages/sdk/src/contracts/searchStreams.ts

```diff
@@ -22,7 +22,6 @@ export interface SearchStreamsOrderBy {
 
 export type SearchStreamsResultItem = {
     id: string
-    userAddress: UserID
```

Comment:
Why removed? Should be replaced with something?

Reply:
It is not needed as we'll eventually just return a collection of `Stream` objects.

Removed also this field from the GraphQL query.